packages/sit-onyx/src/components/OnyxNavBar/modules/OnyxNavButton/NavButtonLayout.vue

```diff
@@ -2,13 +2,17 @@
 // this layout component is only used internally for the nav button component
 // to easily switch between mobile and desktop layout
 import arrowSmallLeft from "@sit-onyx/icons/arrow-small-left.svg?raw";
+import { toRef } from "vue";
+import { useManagedState } from "../../../../composables/useManagedState";
 import { injectI18n } from "../../../../i18n";
 import OnyxButton from "../../../OnyxButton/OnyxButton.vue";
 import OnyxFlyoutMenu from "../OnyxFlyoutMenu/OnyxFlyoutMenu.vue";
 import OnyxNavSeparator from "../OnyxNavSeparator/OnyxNavSeparator.vue";
 import type { OnyxNavButtonProps } from "./types";
 
-const props = defineProps<OnyxNavButtonProps & { isMobile: boolean }>();
+const props = withDefaults(defineProps<OnyxNavButtonProps & { isMobile: boolean }>(), {
+  mobileChildrenOpen: false,
```

Comment:
Why is this false and not the MANAGED_SYMBOL?

Reply:
fixed